scripts/update_docs.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ev
+
+#########################################
+# Only update docs if we are on Travis. #
+#########################################
+if [[ "${TRAVIS_BRANCH}" == "master" ]] && \
+       [[ "${TRAVIS_PULL_REQUEST}" == "false" ]]; then
+  echo "Building new docs on a merged commit."
+elif [[ -n "${TRAVIS_TAG}" ]]; then
+  echo "Building new docs on a tag."
+else
+  echo "No docs to update for a new tag or merged commit on Travis."
+  echo "Verifying docs build successfully."
+  tox -e docs
```

Comment:
@daspecster This means that you don't need `tox -e docs` in `.travis.yml`, since the script also includes it